### DISCUSSION TITLE
Date filter wrong [Sep 07]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3743] [Discord Threads]
> 🙋‍♂️ **MOV**  
#### what i doing wrong?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1011612839049429052/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Can you show what the filter is  
#  
> 💁 **o1lab**  
#### and please paste your project info from right top menu  
#  
> 🙋‍♂️ **MOV**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **pg**   
#### ProjectOnRootDB: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.91.10**  
#  
> 🙋‍♂️ **MOV**  
#### https://drive.google.com/file/d/1K1oJqpLSk2w1Ha5YZfxYXAGR39IgbovU/view?usp=sharing  
#  
> 🙋‍♂️ **MOV**  
#### i gave you answer but forgot reply you  
#  
> 💁 **o1lab**  
#### was able to reproduce the issue in 0.92.4   
#### however in latest develop branch the issue has been resolved  
#  
> 💁 **o1lab**  
#### next build should have the fix  
#  
> 🙋‍♂️ **MOV**  
#### so, what i need to do? just wait next build?  
#  
> 🙋‍♂️ **MOV**  
#### sorry, do you can give me answer?   
#### what we need to do with this problem?   
#### maybe I can try something change in my data base?   
#### Can I get support ?  
#  
> 🙋‍♂️ **MOV**  
#### I still get these errors  
![image0](https://cdn.discordapp.com/attachments/1011612839833772142/1014093840547860550/unknown.png)  
#  
> 💁 **o1lab**  
#### we are in midst of building new UI, hence there is a bit delay  
#  
> 💁 **o1lab**  
#### if you are brave to try our new build - happy to share 😄  
#  
> 💁 **o1lab**  
#### we are almost there  
#  
> 💁 **o1lab**  
#### docker run -d -p 8888:8080 nocodb/nocodb-timely:0.92.4-pr-3364-20220830-0607   
####    
####    
#### one of the builds from today's PR  
#  
> 💁 **o1lab**  
#### happy to take any early feedback  
#  
> 🙋‍♂️ **MOV**  
#### oh!) greate!) I would like to try it!)   
#### Thank you!)  
#  
> 🙋‍♂️ **MOV**  
#### thank you for Ukraine language ❤️ 🇺🇦  
![image0](https://cdn.discordapp.com/attachments/1011612839833772142/1014436693836832798/unknown.png)  
#  
> 💁 **o1lab**  
#### Thank you,    
#### help us improve it too  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1011612839833772142)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
